component/slider/slider.css

```diff
@@ -0,0 +1,20 @@
+.slider-container {
+    width: 50%;
+  }
+  .slider {
+    padding: 0 !important;
```

Comment:
Avoid using `!important`

Reply:
but it was necessary that is why I have used .